src/helpers/share/share-helper.js

```diff
@@ -0,0 +1,19 @@
+import { getUserApi } from '../shared/user-login';
+import { CATALOG_API_BASE } from '../../utilities/constants';
+
+const userApi = getUserApi();
+
+export async function getShareInfo(portfolioId) {
+  //return userApi.shareInfo(portfolioId);
```

Comment:
left over comment

Reply:
- we should use the client method that is in the comment - I left it there as a marker to swith to - I'll add a TODO